code/game/objects/items/kitchen.dm

```diff
@@ -82,6 +102,23 @@
 /obj/item/knife/kitchen
 	name = "kitchen knife"
 	desc = "A general purpose Chef's Knife made by SpaceCook Incorporated. Guaranteed to stay sharp for years to come."
+	var/special_fluff_cooldown = 0
+
+/obj/item/knife/kitchen/attack_self(mob/user)
+	if(special_fluff_cooldown < world.time) //don't spam with it
+		special_fluff_cooldown = world.time + 10 SECONDS
+		do_special_fluff(user)
+	. = ..()
+
+/obj/item/knife/kitchen/proc/do_special_fluff(mob/user)
+	if(prob(90))
+		user.visible_message(span_notice("[user] licks blade of [initial(name)]."), \
+							span_notice("You lick blade of [initial(name)]."))
+	else
+		user.visible_message(span_warning("[user] licks blade of [initial(name)]'s and cuts their tongue!"), \
+							span_danger("You lick blade of [initial(name)] and cut your tongue!"))
+	name = "[initial(name)]"
+	desc = "[initial(desc)] Its blade smells bad."
```

Comment:
Given that the implementations of these procs are exactly the same other than the fluff text, maybe just put the fluff text in a var the subtype can override instead of rewriting the whole proc?